app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/search/ItemSearchScreen.kt

```diff
@@ -110,47 +99,7 @@ fun ItemSearchScreen(
                         modifier = innerModifier,
                     )
                 }
-
-                is ItemSearchState.ViewState.Error -> {
-                    BitwardenErrorContent(
-                        message = viewState.message(),
-                        modifier = innerModifier,
-                    )
-                }
-
-                is ItemSearchState.ViewState.Loading -> {
-                    BitwardenLoadingContent(
-                        modifier = innerModifier,
-                    )
-                }
             }
         }
     }
 }
-
-/**
- * Dialogs displayed within the context of the item search screen.
- */
-@Composable
-private fun ItemSearchDialogs(
```

Comment:
These dialogs were never actually showing, so I removed them. There was quite a bit from this vm/screen that was impossible to ever show, so I removed it out. The resulting vm is simple and nice!